api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java

```diff
@@ -311,13 +311,23 @@ public void setUp() throws IOException {
 
   @Test
   public void testCreateAccount_invalidCaptchaToken() {
+    config.captcha.enableCaptcha = true;
+    createAccountAndDbUserWithAffiliation();
+    createAccountRequest.setCaptchaVerificationToken(WRONG_CAPTCHA_TOKEN);
+
     assertThrows(
-        BadRequestException.class,
```

Comment:
enableCaptcha changes to false in this PR, so by default, this no longer throws.

Split to 2 tests, for each feature flag state